[WIP] Asset Refactor Docs (#590)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/asset/asset.md

### 2.2 New Asset

For new assets, you cannot create the asset record directly from the Asset form. You need to create a Purchase Receipt/Invoice for it.

> When you're buying new assets, an asset ID is created when you make a Purchase Receipt. On submitting the Purchase Receipt for an Asset, you'll see a message like 'Asset AST00003 created'. This will happen only if you've ticked the 'Is Fixed Asset' checkbox when creating the Item.	When you're buying new assets, assets can be created automatically when you make a Purchase Receipt, if you have 'Auto Create Assets on Purchase' ticked in the asset item. On submitting the Purchase Receipt for an Asset, you'll see a message like '5 Assets created for Office Phone'. If you don't want to auto create assets, after Purchase Receipt submission you will have to manually create assets and link the Purchase Receipt to the Asset.

Reviewer: ```suggestion
When you're buying new assets, an asset ID is created when you make a Purchase Receipt. On submitting the Purchase Receipt for an Asset, you'll see a message like 'Asset AST00003 created'. This will happen only if you've ticked the 'Is Fixed Asset' checkbox when creating the Item.

When you're buying new assets, assets can be created automatically when you make a Purchase Receipt, if you have 'Auto Create Assets on Purchase' ticked in the Item master of the Asset. On submitting the Purchase Receipt for an Asset, you'll see a message like '5 Assets created for Office Phone'. If you don't want to auto create assets, after Purchase Receipt submission you will have to manually create assets and link the Purchase Receipt to the Asset.
```

<img class="screenshot" alt="Asset" src="{{docs_base_url}}/assets/img/asset/asset.png">

> After submitting a Purchase Receipt/Invoice for such an Item, in case of auto creation a number of Asset record will be created in the Draft stage. You can then go to the Asset and submit it.

Reviewer: ```suggestion
After submitting a Purchase Receipt/Invoice for such an Item, in case of asset auto creation, a number of Asset records will be created in the Draft stage. You can then go to these Assets and submit them.
```

1. **Custodian**: The employee you're giving/assigning the asset to.
1. **Department**: The department of the Custodian.
1. **Is Existing Asset**: Check if you already own the asset, not purchased recently. The asset could also have been carried forward from the previous Fiscal Year. The existing assets which are partially/fully depreciated can also be created/maintained for future reference.
1. **Opening Accumulated Depreciation**: The accumulated depreciation amount which has already been booked for an existing asset.
1. **Number of Depreciations Booked**: Enter the number of already booked depreciations for an existing asset.
1. **Current Value (After Depreciation)**: In case you are creating a record of an existing asset which has already been partially/fully depreciated, mention the current value of the asset. In case of a new asset, mention the purchase amount or leave it blank.
1. **Next Depreciation Date**: Mention the next depreciation date, even if it is the first one. If the asset is an existing one and depreciation has already been completed, leave it blank.
1. **Calculate Depreciation**: Enable this checkbox to calculate depreciation of Assets.

> 1. **Allow Monthly Depreciation**: Enable this checkbox to distribute depreciation amount of an year into 12 months. Depreciation entries will be made every month on the date provided as Depreciation Start Date. For eg. If Available for use date is 22nd Nov 2019 and depreciation start date is 31st March 2020, first depreciation will be done on 30th Nov 2019 second on 31st Dec 2019 and so on. Amount will be distributed based on days left until next depreciation. 

Reviewer: ```suggestion
1. **Allow Monthly Depreciation**: Enable this checkbox to distribute depreciation amount of an asset into 12 months of the year. Depreciation entries will be made every month on the date provided as Depreciation Start Date. For example, if Available for Use date is 22nd Nov 2019 and depreciation Start Date is 31st March 2020, first depreciation will be done on 30th Nov 2019 second on 31st Dec 2019 and so on. Amount will be distributed based on days left until next depreciation. 
```

## 4. Features

### 4.1 Depreciation

* **Frequency of Depreciation (Months)**: The number of months between depreciations.

> * **Total Number of Depreciations**: The total number of depreciations during the useful life. In case of existing assets which are partially depreciated, mention the number of pending depreciations. For example if you set frequency as 12 months and no. of depreciations as 3, 1 depreciation will be booked every 12 months for 3 years.

Reviewer: ```suggestion
* **Total Number of Depreciations**: The total number of depreciations during the useful life. In case of existing assets which are partially depreciated, mention the number of pending depreciations. For example if you set frequency as 12 months and no. of depreciations as 3, 1 depreciation will be booked every 12 months for 3 years.
```
```suggestion
* **Total Number of Depreciations**: The total number of depreciations during the useful life of the Asset. In case of existing assets which are partially depreciated, mention the number of pending depreciations. For example, if you set frequency as 12 months and no. of depreciations as 3, 1 depreciation will be booked every 12 months for 3 years.
```

> * **Depreciation Method**: There are two options**: Straight Line and Double Declining Balance.

Reviewer: ```suggestion
* **Depreciation Method**: There are two options, Straight Line and Double Declining Balance.
```
